server/weaverbird/steps/rename.py

```diff
@@ -0,0 +1,17 @@
+from typing import List, Tuple
+
+from pandas import DataFrame
+from pydantic import Field
+
+from weaverbird.steps.base import BaseStep
+
+
+class RenameStep(BaseStep):
+    name = Field('rename', const=True)
+    to_rename: List[Tuple[str, str]] = Field(..., alias='toRename')
```

Comment:
:warning: ici j'ai mis `str` plutôt que `ColumnName = Union[str, int, float]` car 1) dans la doc de weaverbird on parle explicitement de strings et 2) le `Union[str, int, float]` fait un cast automatique en `str` de toute façon.

Si on veut vraiment gérer les noms de colonnes `int` et `float` il faut là aussi éviter que pydantic fasse du cast.

Reply:
Yes, tweaking the order is not a solution because if we put `int` before `str`, then `"2020"` will be casted to `2020`.